english/new-exit/new-exit.md

```diff
@@ -1,4 +1,5 @@
 From this exit, you found an entire new world full of mysteries.
 
 All the exits from original world to this world are closed and there is only one way to open it.
-Explore more to find a way to open it.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.